tests/dummy/mirage/serializers/blog-post.js

```diff
@@ -1,5 +1,12 @@
 import { JSONAPISerializer } from 'ember-cli-mirage';
 
 export default JSONAPISerializer.extend({
+  links(blogPost) {
+    return {
+      comments: {
+        related: `/blog-posts/${blogPost.id}/comments`
```

Comment:
@kellyselden, could you please update the path here as well to `/api/blog-posts/${blogPost.id}/comments`?